eng/pipelines/templates/Apex_Tests_On_Windows.yml

```diff
@@ -22,12 +22,6 @@ steps:
   inputs:
     versionSpec: "4.5.0"
 
-- task: PowerShell@1
-  inputs:
-    scriptName: "$(Build.Repository.LocalPath)\\scripts\\utils\\InstallCLIforBuild.ps1"
-    arguments: '$(SDKVersionForBuild)'
-  displayName: "Install .NET 5.0 for build"
-
 - task: PowerShell@1
   displayName: "Define variables"
   inputs:
```

Comment:
This task could be removed from other .yml file as well.

Reply:
The scripts used in task "Install .NET 5.0 for build" could be removed since they're only used in this task.
They're:
https://github.com/NuGet/NuGet.Client/blob/dev-nkolev92-useLatestSDK/scripts/utils/InstallCLIforBuild.ps1
https://github.com/NuGet/NuGet.Client/blob/dev-nkolev92-useLatestSDK/scripts/utils/InstallCLIforBuild.sh

Reply:
Sounds good, I'll clean those up.

Reply:
cleaned them up, tests are passing again.

Reply:
Hi @nkolev92 , looks like the following two installation scrips are not removed:
https://github.com/NuGet/NuGet.Client/blob/dev-nkolev92-useLatestSDK/scripts/utils/InstallCLIforBuild.ps1
https://github.com/NuGet/NuGet.Client/blob/dev-nkolev92-useLatestSDK/scripts/utils/InstallCLIforBuild.sh